docs/getting_started/ibexa_cloud_guide.md

```diff
@@ -0,0 +1,111 @@
+For further protection and easy rollbacks, it also automates your backup procedures.
+This guarantees that the framework developers are working within is adaptable, safe, and responsive to their requirements.
+Thanks to built-in CI/CD features, you can also reduce the need for manual testing and accelerate.
+
+### Digital tool
+
+[[= product_name_cloud =]] is a digital tool - it's a perfect starting step toward your company's digital transformation.
+What's more, [[= product_name_cloud =]] as an end-to-end cloud hosting platform, it's scalable, requires little up-front expenditure, and no ongoing maintenance.
+[[= product_name_cloud =]] as a PaaS solution enables you to start constructing effective digital systems that provide great consumer experiences.
+
+### Disk space
+
+Your [[= product_name_cloud =]] subscription level determines the overall amount of disk space available.
+You can assign disk space to the main app container under the disk key.
+You can also divide the remaining space between additional containers (such as the database) or the search engine in `platform.sh` configuration file, under the specific service definitions.
+
+## Benefits
+
+### Safety and simplicity
+
+Comprehensive data security procedures guarantee that you maintain complete ownership of your client's data, defining where it's stored.
+You can be sure that the safety protocols are compliant with all applicable legislation.
+What's more, all updates to your code and infrastructure are fully auditable. Global CND (Content Delivert Network) is included and fully managed.
+[[= product_name_cloud =]], thanks to grid architecture and extensive [[= product_name_base =]] support, enables easy deployment.
+
+### High availability and compatibility
+
+[[= product_name_cloud =]] is compatible with your choice of public cloud server and support for hosting platforms.
+It's a Git-native development - compatible with Git Flow. You can also integrate with, for example, Bitbucket, GitHub, Gitlab.
+You can instantly clone every branch of both your code and infrastructure.
+If you want to work with services such as MySQL, Kafka, and Elasticsearch, you can simply add them with a line of code.
+What's more, you can run in your chosen cloud, like Microsoft Azure, Orange, Google Cloud Platform, and regional partners.
+
+### Great customer experience
+
+The [[= product_name_cloud =]] as a PaaS solution enables your company to begin developing effective digital systems that provide exceptional client experiences.
+To assist business users and developers in creating user-friendly apps, it also offers a number of extra services, including extensive APIs, workflow, design tools, and more.
+
+### Increase in developer productivity
+
+Thanks to [[= product_name_cloud =]], there is around 40% increase in developer productivity, 15% faster user acceptance testing, and 20% more deployments.
```

Comment:
Do we have sharable resources to confirm those numbers?

Reply:
These numbers are from: https://www.ibexa.co/products/ibexa-cloud
<img width="418" alt="image" src="https://github.com/ibexa/documentation-developer/assets/117284672/2d7ddcba-be88-43c7-871c-e4b1efa5eb49">

Reply:
I'd add a source here as well - I know the style of guides is different from other articles in the doc, but this sentence without a source sounds too marketing-oriented. 

I found this: https://platform.sh/blog/austrian-publishing-giant-kurier-accelerates-development-restores-work-life-balance/ , maybe there is something else we can refer people to.

Reply:
Seem difficult to have a online proof for this.

Side-note, I see Forester argument (hosted by P.sh) added in f0bde0b7c15896ad36fbfd32366a0012b52ca1a4 to "Lower cost" section which is great.